sdks/wasm/Mono.WebAssembly.DebuggerProxy/MonoProxy.cs

```diff
@@ -352,6 +387,7 @@ async Task<bool> OnBreakpointHit (SessionId sessionId, JObject args, Cancellatio
 						var method_token = mono_frame ["method_token"].Value<int> ();
 						var assembly_name = mono_frame ["assembly_name"].Value<string> ();
 
+						var store = await LoadStore (sessionId, token);
```

Comment:
.. this `LoadStore`.

Reply:
Basically we're delaying getting the store until we absolutely, calling LoadStore multiple times is safe